### ISSUE TITLE
Ratgdo Compatibility Thread

### ISSUE THREAD

**Issue by ljmerza** [PaulWieland/ratgdo#148]
Instead of having a new issue for every compatibility question we can make a mega issue for those questions. Out your model and someone can answer in a thread to your comment if it's compatible. Be sure to search for your model before asking for compatibility.

**Comment by ljmerza** [comment]
Is Genie 2055 for ratgdo v2.0? I just got mine right before v2.5 was released so may have to switch.

**Comment by ljmerza** [comment]
would be easier for others to respond here and update there accordingly

**Comment by PaulWieland** [comment]
Yeah. As people respond it can be added to the wiki page for others to reference instead of having to read through a thread.

**Comment by barracksbuilder** [comment]
I have a craftsman 1/2hp Model 139.53964SRT (Date: 10/2002) The control board side has a part number 41A5021-4F. Craftsman are rebadged Chamberlain operners. This unit only has 3 terminals exposed (1 Red and 2 white). I am able to short the 1 red and 1 white to open/close the door. From my reading, it is a Security 1.0 device (Red/Orange learn button) Would the ratgdo 2.5 work? If so what would I need to wire up since it only has 3 screw posts?

**Comment by PaulWieland** [comment]
I'm assuming they just have 1 terminal for ground instead of 2, but can you post a picture? Where are the obstruction sensors connected?

**Comment by barracksbuilder** [comment]
![20231023_104338](https://github.com/PaulWieland/ratgdo/assets/8195097/6d128ce8-af38-4a6a-b933-5a8d86c0e467)

My door control button is in the red #1 screw and white #2 screw, the safety beams are black #3, and white #2

**Comment by Kaldek** [comment]
I have an MJ3800MYQ door opener marked in all documentation as Security+ 2.0 but cannot tell if it is supported.
The wiring is weird.  Pins 1 and 3 are for the "E-Serial" Smart Control Panel 880AUW.  Pin 2 is a dry contact for installing your own button which is odd because the "red wire" is the control wire on all other examples I have seen using ratgdo.

FYI, this is an Australian unit.

![image](https://github.com/PaulWieland/ratgdo/assets/26052829/94c82f0d-8bc7-4ab8-8da4-56b264a85f6a)

![image](https://github.com/PaulWieland/ratgdo/assets/26052829/2546891b-db26-40d1-a580-4bb06d303ee5)

**Comment by PaulWieland** [comment]
> ![20231023_104338](https://user-images.githubusercontent.com/8195097/277531778-6d128ce8-af38-4a6a-b933-5a8d86c0e467.jpg)
> 
> My door control button is in the red #1 screw and white #2 screw, the safety beams are black #3, and white #2.
> 
> Next to the black #3 screw it has a 24v sticker (not sure why it's there). But tested when not obstructed it's 5.88v when there is an obstruction it is 6.24v

You have all three terminals that are needed. White is ground. The newer openers provide two separate ground terminals to avoid having two wires twisted together. 

So wire ratgdo accordingly. red/ctrl -> red, white/gnd -> white, black/obs -> black

**Comment by PaulWieland** [comment]
> I have an MJ3800MYQ door opener marked in all documentation as Security+ 2.0 but cannot tell if it is supported. The wiring is weird. Pins 1 and 3 are for the "E-Serial" Smart Control Panel 880AUW. Pin 2 is a dry contact for installing your own button which is odd because the "red wire" is the control wire on all other examples I have seen using ratgdo.
> 
> FYI, this is an Australian unit.
> 
> ![image](https://user-images.githubusercontent.com/26052829/277542756-94c82f0d-8bc7-4ab8-8da4-56b264a85f6a.png)
> 
> ![image](https://user-images.githubusercontent.com/26052829/277543124-2546891b-db26-40d1-a580-4bb06d303ee5.png)

This is what I'm calling a "ROW" opener - it seems that most openers outside North America have the same terminal setup as yours. Connect terminal 1 to red/ctrl on ratgdo. I need to add this diagram to my wiring instructions page.

**Comment by STDEin** [comment]
I have a Skylink garage door opened (rebranded as Canadian Tire Mastercraft AT-803) that according to the Chamberlain [List of compatible garage door openers for MyQ Garage](https://support.chamberlaingroup.com/s/article/List-of-compatible-garage-door-openers-for-MyQ-Garage-1484145635905) page has partial support "Only able to monitor the status of this Garage Door Opener"

Would that extend to the ratgo? Monitoring but no control, or is it possible that it could control the opener completely?
The opener does not respond to shorted wires to the control pad, but only the the button press on the panel itself.

**Comment by Kaldek** [comment]
> > I have an MJ3800MYQ door opener marked in all documentation as Security+ 2.0 but cannot tell if it is supported. The wiring is weird. Pins 1 and 3 are for the "E-Serial" Smart Control Panel 880AUW. Pin 2 is a dry contact for installing your own button which is odd because the "red wire" is the control wire on all other examples I have seen using ratgdo.
> > FYI, this is an Australian unit.
> > ![image](https://user-images.githubusercontent.com/26052829/277542756-94c82f0d-8bc7-4ab8-8da4-56b264a85f6a.png)
> > ![image](https://user-images.githubusercontent.com/26052829/277543124-2546891b-db26-40d1-a580-4bb06d303ee5.png)
> 
> This is what I'm calling a "ROW" opener - it seems that most openers outside North America have the same terminal setup as yours. Connect terminal 1 to red/ctrl on ratgdo. I need to add this diagram to my wiring instructions page.

Fixed and working.  I just had my circuit design wrong.

**Comment by CrazyCoder** [comment]
Genie 8600IC: https://www.geniecompany.com/files/en/manual/is-isl-ic-h-series.

Dry contact.

Voltage between wall controls terminals (button): 5.8V
Voltage between the open/close switch terminals: 7.85V
Voltage between obstruction sensors terminals: 12.5V

Open/close switch wires/terminals are not exposed. Ground is the screw rail itself. 2 wires from open/close switch contacts go inside the box and need to be spliced for connection. Voltage divider would be needed to be compatible with the current board versions.

![chrome_2023-10-25_17-17-48](https://github.com/PaulWieland/ratgdo/assets/122834/5921257c-b323-4aa7-854c-803a792674bd)


Signal is HIGH when there is an obstruction.
Signal sends PULSE when there is no obstruction.

The pulse looks like this:

![image](https://github.com/PaulWieland/ratgdo/assets/122834/adeb4db9-8253-46bd-b7b2-e35e79f49fa3)

The low pulse length is between 0.4 and 0.5ms.

Will this work with the current pulse detection code if the proper voltage divider is added? What would be the best resistor values for such a divider?

**Comment by mjmcg** [comment]
Hi @PaulWieland - Love this project. I ordered 1 for my Chamberlain, and am wondering if ratgdo will work with my Legacy Overhead door. Here are pictures of it along with the wiring diagram:
![image](https://github.com/PaulWieland/ratgdo/assets/15951940/cbe9e84c-6c49-4076-845f-cb20add8acf9)

![image](https://github.com/PaulWieland/ratgdo/assets/15951940/8b2af86b-569d-420c-bcfa-c0dea33a1b0d)

![image](https://github.com/PaulWieland/ratgdo/assets/15951940/8b75b807-6fd0-48bb-850e-e8fb675c879f)

**Comment by PaulWieland** [comment]
@mjmcg That age chamberlain opener is uncharted territory, we have to do some investigation to see if it has a serial interface or if its just dry contacts.

What wall control do you have? Is it just a door bell or is it multi function?

Worst case scenario you will be able to control it with the dry contact interface. You will have to add two reed switches to act as limit switches for fully open and fully closed.

**Comment by mjmcg** [comment]
Thank you for the quick reply! Here's are pics of the remote, along with the circuit board. It has a light and an open/close button. Happy to add the reed switches. Where would I add them?

![image](https://github.com/PaulWieland/ratgdo/assets/15951940/bc088101-57d7-434c-b09b-aa3913979c85)

![image](https://github.com/PaulWieland/ratgdo/assets/15951940/b3511793-3374-4859-a326-48cb62c18edd)

![image](https://github.com/PaulWieland/ratgdo/assets/15951940/4f5bab78-bdfb-45ca-8791-a770ebe54679)

**Comment by PaulWieland** [comment]
I need to do a write up for this because its going to be a FAQ. 

Get two reed switches. Mount the magnet to the top edge of the door. Mount the "open limit" reed switch on the track so that the switch is closed when the door is fully open. Mount the other reed switch farther down the track so that it is closed when the door is fully closed.

One wire on each reed switch is connected to Ground. The other wires need to be connected to ratgdo terminals Trigger Open & Trigger Close respectively. [See the Genie example on the wiring page.](https://paulwieland.github.io/ratgdo/03_wiring.html)

**Comment by bdb12** [comment]
> I have an MJ3800MYQ door opener marked in all documentation as Security+ 2.0 but cannot tell if it is supported. The wiring is weird. Pins 1 and 3 are for the "E-Serial" Smart Control Panel 880AUW. Pin 2 is a dry contact for installing your own button which is odd because the "red wire" is the control wire on all other examples I have seen using ratgdo.
> 
> FYI, this is an Australian unit.
> 
> ![image](https://user-images.githubusercontent.com/26052829/277542756-94c82f0d-8bc7-4ab8-8da4-56b264a85f6a.png)
> 
> ![image](https://user-images.githubusercontent.com/26052829/277543124-2546891b-db26-40d1-a580-4bb06d303ee5.png)

Hi [Kaldek](https://github.com/Kaldek)

Is there a chance you could post some pics of your install including the Ratgdo unit? I have the same opener as you and just wanted to see what you did. Can I also assume you used a power brick for USB vs finding something on the opener to power the Ratgdo?

Thanks in advance

B

**Comment by PaulWieland** [comment]
@bdb12 the wiring is very straight forward. 

ratgdo - opener
Red/ctrl - 1 eserial 
Wht/gnd - 3 or 4 ground
Blk/obst - 5 ir sensor

the only difference between the 14 terminal ROW openers and NA openers in terms of wiring is the red terminal (it’s green for you). Our openers in NA don’t have a dry contact input.

**Comment by bdb12** [comment]
> @bdb12 the wiring is very straight forward.
> 
> ratgdo - opener Red/ctrl - 1 eserial Wht/gnd - 3 or 4 ground Blk/obst - 5 ir sensor
> 
> the only difference between the 14 terminal ROW openers and NA openers in terms of wiring is the red terminal (it’s green for you). Our openers in NA don’t have a dry contact input.

Thanks so much Paul. Just ordered a Ratdgo about an hour ago. Is the current lead time still 2-3 weeks?

**Comment by Goose66** [comment]
I have a Genie CM7600IC. It has limit switches built-in, and wall control panels with separate light and door buttons, as well as a lock. Is there a secret specification item or class of Genie controller boards that we are looking for for compatibility?

**Comment by PaulWieland** [comment]
@Goose66  as far as I know Genie doors can be controlled with simple dry contact. ratgdo will pull the control line to ground for 1/2 a second which is enough to operate my legacy unit. The limit switches provide feedback so ratgdo knows the four door states.

I'm going to buy a newer genie opener to see what their interface looks like since I might be able to add support for light and lock controls.

**Comment by Goose66** [comment]
I don't think mine is actually "newer," but I would be happy to help. Just need to get my hands on an oscilloscope (and someone who knows how to use it).

**Comment by amferguson** [comment]
I apologize if this is not the correct location to ask this, but does anyone know if this would work for Linear models (labeled as Precision PDS-800)? It does work with my myQ-G0401-ES if that helps at all. I'm not sure how to get a wiring diagram, but here is what the connections coming down from the ceiling look like. 
![280414147-c2edbed9-dfc9-4077-8b2f-6a142fb87e2d](https://github.com/PaulWieland/ratgdo/assets/3903195/9604a294-07c7-4cc1-b28e-64baf8a26010)


Basically in a similar boat to this comment (https://github.com/PaulWieland/ratgdo/issues/159#issuecomment-1787630747) and am trying to flee the myQ ecosystem since it keeps having issues with HomeAssistant. Thanks!

**Comment by richardpowellus** [comment]
![IMG_2596](https://github.com/PaulWieland/ratgdo/assets/1369873/dee9fac9-88bf-4c7c-9f28-ca215928f981)

I think this is a model 8522 which I believe is supported, but I'd love confirmation before I order two kits...

**Comment by PaulWieland** [comment]
> ![IMG_2596](https://user-images.githubusercontent.com/1369873/280424867-dee9fac9-88bf-4c7c-9f28-ca215928f981.JPEG)
> 
> I think this is a model 8522 which I believe is supported, but I'd love confirmation before I order two kits...

Yes its supported. Thats a security + 2.0 (yellow learn button) opener.

**Comment by haffi78** [comment]
I just orderd mine, this is how my Chaimberlain looks like, its an EU model. I assume this will just work i.e. the button today is 1 and 2, and door sensor 2 and 3.
Or is the diagram different for this type ? 
![tempImagew65O4p](https://github.com/PaulWieland/ratgdo/assets/4826382/d405a6b6-8bb8-43c8-a877-bb6d81868124)
 
Thanks !

**Comment by dcode** [comment]
I've got two 8500W openers, which are the wall mounted jack openers with WiFi and battery builtin. These openers have the usual beam obstruction sensor as well as some sort of reed switch up top, never really figured out what that was for (one door is missing the paddle so clearly useless).

It also has a lock (looks like headers for possibly 2 locks) on the side of the channel and wirelessly pairs with a light.

I feel like these were the upgraded from 8500C and use Security+ 2.0, but sounds like the v2.5 boards will work even if it's 1.0.

If I get it all wired up correctly, does the board report the status of the lights and lock as well?


![PXL_20231106_150349367.jpg](https://github.com/PaulWieland/ratgdo/assets/171574/bcd12bdf-9c32-4f0e-bafa-c71b1de36e10)

![PXL_20231106_150538159.jpg](https://github.com/PaulWieland/ratgdo/assets/171574/7bc8a6c9-6c13-4bfd-92f6-195a5f63c3ed)

![PXL_20231106_150503165.jpg](https://github.com/PaulWieland/ratgdo/assets/171574/8862152c-903f-48bf-9869-ce7cc37e4d17)

**Comment by PaulWieland** [comment]
> If I get it all wired up correctly, does the board report the status of the lights and lock as well?

It reports the status of the wireless lockout. If the mechanical lock status is reported on the serial line we can report that too, but it would take some debugging from you once you have a ratgdo installed so that we can identify if the lock status is communicated and if so, what the data looks like.

**Comment by PaulWieland** [comment]
> I just orderd mine, this is how my Chaimberlain looks like, its an EU model. I assume this will just work i.e. the button today is 1 and 2, and door sensor 2 and 3. Or is the diagram different for this type ? 

The ROW openers have a different terminal for the serial data. Your manual should show one of them as being "eserial" - usually its the first terminal which is sometimes labeled `#0` and sometimes `#1`. See https://github.com/PaulWieland/ratgdo/wiki where I mention the openers with 14 terminals.

**Comment by MikeMcGlinch** [comment]
I have 2 Chamberlain GDOs.  Do I need 1 of your devices for each Opener?  Or will one device operate both?

**Comment by rlowens** [comment]
> I have 2 Chamberlain GDOs. Do I need 1 of your devices for each Opener? Or will one device operate both?

You need one for each door. @PaulWieland maybe add this to the FAQ? I've seen this question at least 3 times in various places.

**Comment by larryopoly** [comment]
> > ![20231023_104338](https://user-images.githubusercontent.com/8195097/277531778-6d128ce8-af38-4a6a-b933-5a8d86c0e467.jpg)
> > My door control button is in the red #1 screw and white #2 screw, the safety beams are black #3, and white #2.
> > Next to the black #3 screw it has a 24v sticker (not sure why it's there). But tested when not obstructed it's 5.88v when there is an obstruction it is 6.24v
> 
> You have all three terminals that are needed. White is ground. The newer openers provide two separate ground terminals to avoid having two wires twisted together.
> 
> So wire ratgdo accordingly. red/ctrl -> red, white/gnd -> white, black/obs -> black

@barracksbuilder, I seem have the same model, just a year older (41a5021-3E). Mind reporting back if/when you're up and running?

**Comment by thickconfusion** [comment]
Compatibility with Genie 2028?

**Comment by tigeinc** [comment]
Hello, have purchased a 2.5 board, hoping sound out compatibility with a Merlin SilentDrive Elite MR865MYQ.
This GDO and its variants are currently sold and seem to be fairly common in Australia and New Zealand.
[MR865MYQ manual](https://p.widencdn.net/ynlory/MR865MYQSilentDrive-Elite-MyQ-Manual-114A4903C)
<img width="1209" alt="terminal block diagram" src="https://github.com/PaulWieland/ratgdo/assets/32529403/35e12476-96c6-46f6-801b-11a35b578cf2">
<img width="1218" alt="wiring diagram" src="https://github.com/PaulWieland/ratgdo/assets/32529403/69e83f20-9543-44a5-8a94-823f53c7647f">
<img width="551" alt="control buttons" src="https://github.com/PaulWieland/ratgdo/assets/32529403/5941cbdc-4020-442e-8c9a-1029bce30a7e">
<img width="515" alt="terminal block photo" src="https://github.com/PaulWieland/ratgdo/assets/32529403/e14c8701-6b92-4533-901b-1385932f7dda">
It uses Security2.0+, but has a different learning/program button arrangement.
No explicit reference to eserial in the manual or documentation.

Wondering if anyone has installed with this unit or encountered similar?

**Comment by NukeThemTillTheyGlow** [comment]
Before ordering what appears to be 3 v2.5 boards, I thought I would confirm the details of the 3 openers I have:
**Manufacture	Model	                        Model number   Button**
Liftmaster	Professional Formula 1	2280-267	          Red
Liftmaster	8550W	                        8550W	          Yellow
Liftmaster		                                87504-267	  Yellow

**Comment by shanelord01** [comment]
**Working - Australian Merlin Door Opener**

Model: Merlin MS105MYQ
Country: Australia
ratgdo version: 2.5i
s/w: ESPHome

Connectors:
Merlin 0 - eserial (red/Ctrl)
Merlin 2 - ground (white/GND)

**Comment by pixelrazor** [comment]
Sharing my experience hooking up my Genie 1028:

Hooking the voltage divider to the other side of the obstruction sensor was putting out a lower voltage than hooking it to the other ground. It shouldn't make a difference but it did. Could also be due to bad contact with the twisted wires. Either way, the obstruction sensing seems a bit flaky - I'll try to collect more usage to see. I have reed switches on the way for the door state, but opening and closing works fine. The voltage divider is with two 10k resistors.

![PXL_20231115_003636587.jpg](https://github.com/PaulWieland/ratgdo/assets/24706952/a6979436-27fd-41ee-9337-62e026e26b9c)

**Comment by PaulWieland** [comment]
I need to write an auto sensing frequency calculator for the obstruction input. Not all doors use the same frequency. I think some (if not most) genie openers use a much slower pulse than the chamberlain ones. Sharing my experience hooking up my Genie 1028:
Hooking the voltage divider to the other side of the obstruction sensor was putting out a lower voltage than hooking it to the other ground. It shouldn't make a difference but it did. Could also be due to bad contact with the twisted wires. Either way, the obstruction sensing seems a bit flaky - I'll try to collect more usage to see. I have reed switches on the way for the door state, but opening and closing works fine. The voltage divider is with two 10k resistors.

**Comment by CrazyCoder** [comment]
> I think some (if not most) genie openers use a much slower pulse than the chamberlain ones

At least my model has faster pulses than ratgdo is expecting: https://github.com/PaulWieland/ratgdo/issues/168.

**Comment by MRobi1** [comment]
My dad's got an old Liftmaster opener. I'm unsure of the model number at the moment, but it's got a green learn button. It's definitely a dry contact opener so I know I could get the open/close to work. I'm just wondering if it would be able to read the door position off the motor like it does with the newer openers
